test/gtest/midend_test.cpp

```diff
@@ -56,28 +58,33 @@ TEST(midend, convertEnums_pass) {
     ASSERT_NE(nullptr, pgm);
 }
 
+// use enum before declaration should fail
 TEST(midend, convertEnums_used_before_declare) {
-    string program =
+    std::string program =
         "const bool a = E.A == E.B;\n"
         "enum E { A, B, C, D };\n";
     const IR::P4Program* pgm = parse_string(program);
     ASSERT_NE(nullptr, pgm);
+
     ReferenceMap  refMap;
     TypeMap       typeMap;
     auto convertEnums = new P4::ConvertEnums(&refMap, &typeMap, new EnumOn32Bits());
     PassManager passes = {
         convertEnums
     };
     pgm = pgm->apply(passes);
+    // expected pgm == nullptr
```

Comment:
That's pretty clear even without the comment. =) The comment you added on line 61 is enough on its own, I think, though it might be clearer to move it down here.